docker-compose.yaml

```diff
@@ -17,6 +17,7 @@ services:
     build:
       context: .
       target: release  # build the release stage
+    platform: linux/amd64
```

Comment:
I'm assuming this can still be run on a Mac M-Series device using ARM? It just won't be native and as fast?